serviceUsage.html

```diff
@@ -345,7 +362,67 @@ <h2>
                 <td>When specified message protocol is not loaded</td>
             </tr>
         </table>
-
+		<h2>
+            <a id="examples" class="anchor" href="#examples" aria-hidden="true">
+                <span aria-hidden="true" class="octicon octicon-link"></span>
+            </a>
+        </h2>
+		<h2> Lookups </h2>
+		<p> Presently, to generate Eiffel messages, Eiffel REMREM-Generate is being used. This process requires input mapping exactly one-to-one to Eiffel message Schema.</p>
+		<p> The "Link" field of Eiffel Message requires UUIDs of earlier sent events to link to. But these UUIDs are not always available .</p>
+		<p> In order to resolve this , we need to provide LOOKUPS in Links field ,which will have eventType and Properties. eventType will contain type of event.</p>
+		<p> Properties will contain any field of event mentioned in eventType or RepoUri or artifact identity (Purl identity including version) to Eiffel ,</p>
+		<p> which will make sure it is linked to the correct events.</p>
+		<p> The Parameters specified should be enough to replace the listed event id as input </p>
+		<h5> Templates for LOOKUPS : </h5>
+		<pre>
+		"links":[
+					{
+			"type": "ACTIVITY_EXECUTION",
+			"%lookup%": {
+			"eventType": "EiffelArtifactCreatedEvent",
+			"properties": [
+				{ “data.identifier": "some purl identifier defining the artifact" }]
+					}
+				}]
+		</pre>
+		<h5> Examples for Lookups:</h5>
+		<h6> Example 1:</h6>
+		<pre>
+		"links": [
+			"type": “FLOW_CONTEXT",​
+			"%lookup%": {​
+			"eventType": "EiffelArtifactCreatedEvent",​
```

Comment:
same as above comment. You should look in the EiffelFlowContextDefinedEvent if you need to add a link to FLOW_CONTEXT.